examples/NavigationPlayground/js/SimpleTabs.js

```diff
@@ -11,21 +11,20 @@ import {
 } from 'react-native';
 import {
   TabNavigator,
+  withNavigation,
 } from 'react-navigation';
 import Ionicons from 'react-native-vector-icons/Ionicons';
 import SampleText from './SampleText';
 
+const ButtonWithNavigation = withNavigation(({ navigation, to, ...rest}) => {
```

Comment:
I think we should keep this out of the simple example, but add it to multiple of the other examples, and/or add new examples. Lets do our best to not pollute the simple examples, and when it makes sense, make new examples rather than making the existing ones endlessly more complicated.